languages/en.ini

```diff
@@ -1184,7 +1184,12 @@ See also = "See also"
 see_all_ellipsis = "see all…"
 Select this record = "Select this record"
 Select your carrier = "Select your carrier"
-select_page = "Select Page"
+select_item_checked_out_renew = "Select item for renewing"
+select_item_hold_cancel = "Select item for canceling holds or recalls"
+select_item_ill_request_cancel = "Select item for canceling interlibrary loan requests"
+select_item_storage_retrieval_request_cancel = "Select item for canceling storage retrieval requests"
+select_page = "Select all entries of the page"
+select_page_cart = "Select all entries of the Book Bag"
```

Comment:
I'm not sure if I like "entries of the" in these two language strings. How would you feel about:

```
select_page = "Select all items on the page"
select_page_cart = "Select all items in the Book Bag"
```

I think on/in works better than of, and item seems a little more general purpose (and shorter) than entry as the noun.

Reply:
That's fine with me. I changed "entries" to "items". 
Also restored `select_page = "Select Page"` and added `select_all_on_page = "Select all items of the page"`

Reply:
I still think `items on the page` makes more sense in English than `items of the page`. Do you mind changing that? (Sorry to be so picky!).

If select_page is no longer used, we might be able to get rid of it, though perhaps it is worth retaining in case we are unable to get new translations for select_all_on_page for some languages -- in those cases, the existing select_page translation might be better than nothing.

Reply:
Thanks for the hint, I missed that. Now it is: `select_all_on_page = "Select all items on the page"`

Reply:
`select_page` is still used in other templates

Reply:
Thanks @demiankatz !
I changed `select_page` to `select_all_on_page` now in `holds/list.phtml` and `myresearch/checkedout.phtml`

Reply:
Thanks! I think we have a couple of other threads awaiting input from @RLangeUni, so I'll wait and see how that develops before reviewing this further. Let me know if I can help with anything else in the meantime, or if you'd like to try to get some parts of this merged while we wait for others to be resolved.

Reply:
Sorry for my late reply. I'm trying to get back into pull request today and next week after my long absence.